scripts/run_backend_tests.py

```diff
@@ -390,6 +427,13 @@ def check_test_results(tasks, task_to_taskspec, generate_coverage_report):
                 tests_run_regex_match = re.search(
                     r'Ran ([0-9]+) tests? in ([0-9\.]+)s',
                     task.task_results[0].get_report()[0])
+                if not tests_run_regex_match:
+                    # There was an error, and the tests did not run
+                    # (The error message did not match
+                    # `tests_run_regex_match`).
```

Comment:
I think the comment is a bit redundant now we have a message in the exception.

Reply:
removed the comment